From 7f1583ac861b74234f86a4098768ed6b04e2b99d Mon Sep 17 00:00:00 2001
From: Felix Yan <felixonmars@archlinux.org>
Date: Mon, 27 May 2019 16:02:01 +0800
Subject: [PATCH] Use Brotli instead of brotlipy

brotlipy is stuck at brotli 0.6 and upstream is inactive. Let's switch
to the official binding which is up-to-date and has same interfaces.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index afbef58..a25ce45 100644
--- a/setup.py
+++ b/setup.py
@@ -35,7 +35,7 @@ setup(
     packages=find_packages(),
     include_package_data = True, # include files listed in MANIFEST.in
     install_requires=[
-        'Flask', 'MarkupSafe', 'decorator', 'itsdangerous', 'six', 'brotlipy',
+        'Flask', 'MarkupSafe', 'decorator', 'itsdangerous', 'six', 'Brotli',
         'raven[flask]', 'werkzeug>=0.14.1'
     ],
 )
-- 
2.23.0

